`make check` broken if MultiTerm is disabled

```console
Making check in po
make[1]: Entering directory `/home/travis/build/b4n/geany-plugins/_build/po'
make[1]: *** No rule to make target `../../multiterm/src/config.c', needed by `geany-plugins.pot'.  Stop.
make[1]: Leaving directory `/home/travis/build/b4n/geany-plugins/_build/po'
make: *** [check-recursive] Error 1
```

71fa02ef2f0f846d42a3082136fd236b0da49ab6 listed the *.c* in *POTFILES.in*, but those are only generated when MultiTerm is actually built.

--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -236,12 +236,6 @@ multiterm/src/plugin.vala
 multiterm/src/shell-config.vala
 multiterm/src/tab-label.vala
 multiterm/src/terminal.vala
-multiterm/src/config.c
-multiterm/src/context-menu.c
-multiterm/src/notebook.c
-multiterm/src/plugin.c
-multiterm/src/tab-label.c
-multiterm/src/terminal.c
 
 # Overview
 overview/overview/overviewcolor.c

--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -3,5 +3,13 @@
 #geanyvc
 geanyvc/src/commit.glade
 
+# Multiterm
+multiterm/src/config.c
+multiterm/src/context-menu.c
+multiterm/src/notebook.c
+multiterm/src/plugin.c
+multiterm/src/tab-label.c
+multiterm/src/terminal.c
+
 # WebHelper
 webhelper/src/gwh-enum-types.c

